### PR TITLE
akumuli: new port

### PR DESCRIPTION
**Pull request by ierofant**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: databases/Akumuli/Portfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
```
Comment (cjones051073) [inline databases/Akumuli/Portfile:6]: The cxx11 portgroup is deprecated. Please replace with

 compiler.cxx_standard 2011

**Comment by cjones051073** [comment]
Port is failing to build, see the test results.

https://paste.z0k.xyz/1319f7c0a554

Looks like a missing port dependency

File: databases/Akumuli/Portfile (hunk 1 above)
Comment (ierofant) [reply databases/Akumuli/Portfile]: Done

File: databases/Akumuli/Portfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    size    551148
+
+depends_lib-append  port:boost \
+                    port:jemalloc \
+                    port:sqlite3 \
+                    port:libmicrohttpd \
+                    port:apr \
+                    port:apr-util \
+                    port:muparser \
+                    port:log4cxx
```
Comment (reneeotten) [inline databases/Akumuli/Portfile:29]: the build system doesn't find `log4cxx` and `muparser` even though you've added the dependencies here:
`-- NOT Found log4cxx, disabling it`
`-- Not found muParser`

**Pull request merged**